pp/integrations/plugin-packs/procedures/network-keysight-nvos-restapi.md

```diff
@@ -201,103 +193,13 @@ apt install centreon-plugin-network-keysight-nvos-restapi
 3. Apply the **Net-Keysight-Nvos-Restapi-custom** template to the host.
 4. Once the template is applied, fill in the corresponding macros. Some macros are mandatory.
 
-| Mandatory   | Macro                       | Description                                                                       | Default |
-|:------------|:----------------------------|:----------------------------------------------------------------------------------|:--------|
-| x           | KEYSIGHTNVOSAPIPASSWORD     | API password                                                                      |         |
-|             | KEYSIGHTNVOSAPIPORT         | Port used                                                                         | 8000    |
-|             | KEYSIGHTNVOSAPIPROTO        | Specify https if needed                                                           | https   |
-| x           | KEYSIGHTNVOSAPIUSERNAME     | API username                                                                      |         |
-|             | KEYSIGHTNVOSAPIEXTRAOPTIONS | Any extra option you may want to add to every command line (eg. a --verbose flag) |         |
-
-
-### Service 
```

Comment:
Same question regarding the "Service" section

Reply:
because it was done with the script version before